examples/dockerfile/BUILD.bazel

```diff
@@ -87,3 +88,95 @@ assert_oci_image_command(
            ||     ||
 """,
 )
+
+# This tar can be arranged in arbitrary ways.
+# For example, one could grab pkg_files from elsewhere in the workspace.
+# Anything in the tar can be used by a Dockerfile ADD or COPY.
+mtree_spec(
+    name = "buildx_context_tree",
+    srcs = [
+        "context.Dockerfile",
+        "requirements.txt",
+    ] + glob([
+        "src/*",
+    ]),
+)
+
+mtree_mutate(
+    name = "buildx_context_tree_prefixed",
+    mtree = ":buildx_context_tree",
+    strip_prefix = package_name(),
+    package_dir = "/app",
```

Comment:
you need a `./app` to fix the issue.

Reply:
Thanks - this works

One thing I found a little surprising is that I need to add the `srcs` for both the `mtree_spec` and the `tar`. 

Is this expected? Am I using it correctly?